test/e2e/storage/vsphere/vsphere_utils.go

```diff
@@ -69,42 +69,37 @@ const (
 // Wait until vsphere volumes are detached from the list of nodes or time out after 5 minutes
 func waitForVSphereDisksToDetach(nodeVolumes map[string][]string) error {
 	var (
-		err            error
-		disksAttached  = true
 		detachTimeout  = 5 * time.Minute
 		detachPollTime = 10 * time.Second
 	)
-	err = wait.Poll(detachPollTime, detachTimeout, func() (bool, error) {
+	err := wait.Poll(detachPollTime, detachTimeout, func() (bool, error) {
```

Comment:
nit: waitErr for style consistency - also no shadowing without a shadow of a doubt

Reply:
+1